<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HJ212Dialog</class>
 <widget class="QDialog" name="HJ212Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>895</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HJ212-2017服务器端</string>
  </property>
  <widget class="QGroupBox" name="hj212groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>711</width>
     <height>791</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>HJ212报文解析显示</string>
   </property>
   <widget class="QLabel" name="labelFlag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flag：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MN：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFlag">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditST">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelST">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ST：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CN：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>701</width>
      <height>481</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>请求时间QN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>数据时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>因子编码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>监测值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>数据标记</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监测数据：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelQN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>QN：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="tcpServerGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>TCP服务器</string>
   </property>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>31</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>监听端口:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListen">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>监听</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="logGroupBox">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>511</width>
     <height>761</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>接收日志报文信息</string>
   </property>
   <widget class="QTextEdit" name="textEditRead">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
